docs/content/preview/migrate/reference/yb-voyager-cli.md

```diff
@@ -291,19 +479,27 @@ Specifies the username of the target database.
 
 Specifies the password of the target database.
 
-If the password contains special characters, enclose it in single quotes.
+If you don't provide a password via the CLI during any migration phase, yb-voyager will prompt you at runtime for a password.
+
+If the password contains special characters that are interpreted by the shell (for example, # and $), enclose it in single quotes.
 
 ### --target-db-name
 
 Specifies the name of the target database.
 
+Default: yugabyte
+
 ### --target-db-schema
 
 Specifies the schema of the target database. Applicable only for MySQL and Oracle.
 
 ### --parallel-jobs
 
-Specifies the count to increase the number of connections.
+Specifies the number of parallel COPY commands issued to the target database.
+
+If yb-voyager can determine the total number of cores `N` in the target YugabyteDB cluster, it will use `N/2` as the default value of `--parallel-jobs`. Otherwise, it will default to twice the number of nodes in the cluster.
```

Comment:
```suggestion
Default: If yb-voyager can determine the total number of cores `N` in the target YugabyteDB cluster, it uses `N/2` as the default. Otherwise, it defaults to twice the number of nodes in the cluster.
```